packages/app/lib/ui/widgets/sighting_form.dart

```diff
@@ -1,149 +1,143 @@
-                    borderRadius: BorderRadius.circular(12),
-                    validator: FormBuilderValidators.compose(
-                        [FormBuilderValidators.required()]),
-                    items: speciesOptions
-                        .map((species) => DropdownMenuItem(
-                              alignment: AlignmentDirectional.centerStart,
-                              value: species,
-                              child: Text(species),
-                            ))
-                        .toList(),
-                    onChanged: (val) {
-                      setState(() {
-                        _speciesHasError = !(_formKey
-                                .currentState?.fields['species']
-                                ?.validate() ??
-                            false);
-                      });
-                    },
-                    valueTransformer: (val) => val?.toString(),
-                  ),
-                )),
-                const SizedBox(height: 15),
-                MeliCard(
-                    child: Container(
-                  margin: EdgeInsets.all(4),
-                  child: FormBuilderTextField(
-                    autovalidateMode: AutovalidateMode.always,
-                    name: 'popular_name',
-                    decoration: InputDecoration(
-                      border: OutlineInputBorder(
-                          borderRadius: BorderRadius.circular(12)),
-                      suffixIcon: _popularNameHasError
-                          ? const Icon(Icons.error, color: Colors.red)
-                          : const Icon(Icons.check, color: Colors.green),
-                    ),
-                    onChanged: (val) {
-                      setState(() {
-                        _popularNameHasError = !(_formKey
-                                .currentState?.fields['popular_name']
-                                ?.validate() ??
-                            false);
-                      });
-                    },
-                    // valueTransformer: (text) => num.tryParse(text),
-                    validator: FormBuilderValidators.compose([
-                      FormBuilderValidators.required(),
-                      FormBuilderValidators.max(70),
-                    ]),
-                    textInputAction: TextInputAction.next,
-                  ),
-                )),
-              ],
-            ),
+                    SightingImagesCarousel()
+                  ],
+                ))));
+  }
+}
+
+class SightingImagesCarousel extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final cameraImageProvider = MeliCameraProviderInherited.of(context);
+    final showDelete = !cameraImageProvider.isEmpty();
+    final images = !cameraImageProvider.isEmpty()
+        ? cameraImageProvider.getAll().map((file) {
+            return Image.file(file);
+          })
+        : [Image.asset(PLACEHOLDER_IMG)];
```

Comment:
Does it need to be so closely tied to the camera functionality? I would assume that the carousel is just a "simple" widget displaying any image it gets from the outside, not needing to know where they came from or how they are managed.